src/Client.php

```diff
@@ -117,22 +129,25 @@ public function getVersion(bool $without_hash = true): string
     /**
      * {@inheritdoc}
      */
-    public function devToken(string $username,
-                             string $password,
-                             bool $is_hash = false,
-                             ?DateTime $date_from = null,
-                             int $age = 60): DevTokenResponse
+    public function devToken(DevTokenParams $params): DevTokenResponse
     {
+        $date_time_immutable = $params->getDateFrom() ?? new \DateTimeImmutable;
+
+        // Convert \DateTimeImmutable to \DateTime with timezone
+        // @notice We can use DateTime::createFromImmutable() when the PHP 7.2 support is removed
+        $date_time = new \DateTime('now', $date_time_immutable->getTimezone());
+        $date_time->setTimestamp($date_time_immutable->getTimestamp());
+
```

Comment:
Out of scope. Need remove.

Reply:
Rollback changes with DateTimeImmutable  932cac9bf91aa218d65c4d2f598e084367459b2c